packages/xo-web/src/common/report-bug-button.js

```diff
@@ -24,32 +29,78 @@ export const reportBug = ({ formatMessage, message, title }) => {
   )
 }
 
-const ReportBugButton = ({
-  formatMessage,
+const SUPPORT_PANEL_URL = `${window.location.origin}/supportPanel/create/ticket`
+
+const reportBugWithFiles = ({
+  files,
+  formatMessage = identity,
   message,
-  rowButton,
   title,
-  ...props
 }) => {
-  const Button = rowButton ? ActionRowButton : ActionButton
-  return (
-    <Button
-      {...props}
-      data-formatMessage={formatMessage}
-      data-message={message}
-      data-title={title}
-      handler={reportBug}
-      icon='bug'
-      tooltip={_('reportBug')}
-    />
-  )
+  const { FormData, open } = window
+
+  const formData = new FormData()
+  formData.append('title', title)
+  if (message !== undefined) {
+    formData.append('message', formatMessage(message))
+  }
+  files.forEach(({ file, name }) => {
+    formData.append('attachments', file, name)
+  })
```

Comment:
I think it would be nice to always add a file describing the environment (e.g. installed packages).